tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -353,7 +354,6 @@ def test_failover_control_pod_kubectl(self):
         """
         LOGGER.info("STARTED: Verify IOs before and after control pod fails over, verify control "
```

Comment:
We should mention that this test is about tainting control node with no schedule option in test name.